.github/workflows/release.yml

```diff
@@ -53,6 +53,29 @@ jobs:
         echo "::set-output name=tag_patch::${TAG_PATCH}"
         echo "::set-output name=tag_special::${TAG_SPECIAL}"
 
+    - name: Validate commits and tag signatures
+      run: |
+        
+        # Import Matt Holt's key
+        curl 'https://github.com/mholt.gpg' | gpg --import
+
+        # first we verify the tag, which are only accepted if signed by Matt's key
+        git verify-tag "${{ steps.vars.outputs.version_tag }}" || exit 1
+
+        # Import Github's key, which is used for web merges
+        curl 'https://github.com/web-flow.gpg' | gpg --import
```

Comment:
Hmm. So this means that the release will be rejected if anyone else committed straight to master? I'm not sure if this is an assumption that can be kept forever. Matt is a BDFL basically but I'm not sure he'll always be the only one to commit directly? I guess we could make that a policy though. But it's a decision.

Reply:
So like, what happens if a PR is merged but not squashed (fast-forward merge), I figure the commits remain intact, not signed by github. So if we ever don't use the squash merge button, it'll make the release fail. 🤔

Reply:
Good points. The commits do stay intact and not be signed by Github. We can assume [Linus' stance](http://git.661346.n2.nabble.com/GPG-signing-for-git-commit-td2582986.html) by trusting the entire tree if the tag or the HEAD is signed by a trusted committer. Which should be our preference?

Reply:
I think just looking at the tag is enough. It means that Matt is 👍 on the contents.

Reply:
Done.